### ISSUE TITLE
Include info for Mac users in Wiki and/or comments

### ISSUE THREAD

**Issue by jlamson** [Gremious/StS-DefaultModBase#70]
# tl;dr Help out Mac users please :) 

I'm moving through the Wiki to try modding, and there's a few tweaks for Mac users in the [Adding Dependencies](https://github.com/Gremious/StS-DefaultModBase/wiki/Step-1:-Setting-Up-Your-Integrated-Development-Environment-and-the-Default-Mod-Base#adding-dependencies)

* For the `Steam.path`, mac users should look in `/Users/{yourUser}/Library/Application Support/Steam/steamapps/`
* All the `systemPath` values for the `modthespire`, `basemod`, and `StSLib` worked correctly
* For `desktop-1.0.jar`, the path provided won't work. On mac, everything's bundled in `SlayTheSpire.app`. If you right click that file in finder and select "Show Package Contents", you can see there's a buncha stuff within the app. The path for mac users should be `${Steam.path}/common/SlayTheSpire/SlayTheSpire.app/Contents/Resources/desktop-1.0.jar`
* For the package configuration, the `tofile` in needs to be in the `Resources/mods` folder: `<copy file="target/${project.artifactId}.jar" tofile="${Steam.path}/common/SlayTheSpire/SlayTheSpire.app/Contents/Resources/mods/${project.artifactId}.jar"/>`

The above could be added as additional notes in the wiki page linked above, or perhaps as commented out lines like the local `../lib/` paths are

Edit: Found another one that maybe is clear and I just didn't get it. In the [Setting up debugging](https://github.com/Gremious/StS-DefaultModBase/wiki/Step-3:-Packaging-and-Playing-the-Default;-Writing-Your-First-Mod!#setting-up-debugging) step, the "intstall directory" is also buried in the `.app` folder. I had to set it to `pathToSteam/common/SlayTheSpire/SlayTheSpire.app/Contents/Resources`, as that had the `jre/bin/java` path in it that MtS needs to run.
